Mathlib/Topology/Basic.lean

```diff
@@ -1229,6 +1229,13 @@ theorem Dense.inter_nhds_nonempty (hs : Dense s) (ht : t ∈ 𝓝 x) :
   let ⟨U, hsub, ho, hx⟩ := mem_nhds_iff.1 ht
   (hs.inter_open_nonempty U ho ⟨x, hx⟩).mono fun _y hy => ⟨hy.2, hsub hy.1⟩
 
+/-- The union of a set and the complement of its closure is a dense set. -/
+theorem union_compl_closure_dense {s : Set X} :
+    Dense (s ∪ (closure s)ᶜ) := by
```

Comment:
Is it the same as the recently added `coborder`?

Reply:
Looks like it.